{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "f3917f9c_11291e8f",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 31957
      },
      "writtenOn": "2024-03-06T00:42:09Z",
      "side": 1,
      "message": "Looks like the failure is unrelated to the patch?",
      "revId": "76c8790d0eb16b7f4635623ca0ab06223b629b5d",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    }
  ]
}